pkg/proxy/endpoints.go

```diff
@@ -349,8 +349,8 @@ func getLastChangeTriggerTime(annotations map[string]string) time.Time {
 	}
 	val, err := time.Parse(time.RFC3339Nano, annotations[v1.EndpointsLastChangeTriggerTime])
 	if err != nil {
-		klog.Warningf("Error while parsing EndpointsLastChangeTriggerTimeAnnotation: '%s'. Error is %v",
-			annotations[v1.EndpointsLastChangeTriggerTime], err)
+		klog.ErrorS(err, "Error while parsing EndpointsLastChangeTriggerTimeAnnotation",
+			"endpointsLastChangeTriggerTimeAnnotation", annotations[v1.EndpointsLastChangeTriggerTime])
```

Comment:
https://github.com/kubernetes/kubernetes/pull/104891/files#r709232121

Reply:
Same as above

Reply:
I would say Info too, it doesn't impact kube-proxy behavior and having this erroring may give a false sensation something is wrong

Reply:
Understood, makes sense, thanks!